client/my-sites/checkout/composite-checkout/components/refund-policies.tsx

```diff
@@ -206,6 +236,13 @@ function RefundPolicyItem( { refundPolicy }: { refundPolicy: RefundPolicy } ) {
 			);
 			break;
 
+		case RefundPolicy.GiftBiennialPurchase:
+			text = translate(
+				'You understand that gift {{refundsSupportPage}}refunds{{/refundsSupportPage}} are limited to 14 days after purchase for products with biennial subscriptions.',
```

Comment:
This should probably say "two year" rather than "biennial", to match the others.

Reply:
Done.